versions.mk

```diff
@@ -24,7 +24,12 @@ vVERSION := v$(VERSION:v%=%)
 GOLANG_VERSION ?= 1.20.4
 CUDA_VERSION ?= 11.8.0
 
+CLIENT_GEN_VERSION ?= v0.26.1
+CONTROLLER_GEN_VERSION ?= v0.9.2
+GOLANGCI_LINT_VERSION ?= v1.52.0
+MOQ_VERSION ?= v0.3.4
+
 BUILDIMAGE_TAG ?= devel-go$(GOLANG_VERSION)
-BUILDIMAGE ?=  ghcr.io/nvidia/k8s-test-infra:$(BUILDIMAGE_TAG)
+BUILDIMAGE ?=  ghcr.io/nvidia/$(DRIVER_NAME):$(BUILDIMAGE_TAG)
```

Comment:
I would make a distinction here between the BUILDIMAGE and the BASE_BUILDIMAGE.

Reply:
It's still unclear how this PR allows me to use the `k8s-infra` image by default, but gives me the option to use the local devel image if I want.

Reply:
now you can if desired
```bash
BUILDIMAGE=kklues/dra-demo:devel make build-image 
BUILDIMAGE=kklues/dra-demo:devel make docker-*
```
if BUILDIMAGE is not set, it will default to the `k8s-test-infra` one